ceph.spec.in

```diff
@@ -967,7 +967,8 @@ rm -rf %{buildroot}
 %{_unitdir}/ceph.target
 %{python_sitelib}/ceph_detect_init*
 %{python_sitelib}/ceph_disk*
-%{python_sitelib}/ceph_volume*
```

Comment:
`%dir %{python_sitelib}/ceph_volume`

Otherwise build fails with 

```
[ 4618s] ceph-base-12.1.2+git.1501876501.4c219a30d6-1.1.x86_64.rpm: directories not owned by a package:
[ 4618s]  - /usr/lib/python2.7/site-packages/ceph_volume
```